docs/sources/tempo/release-notes/v2-1.md

```diff
@@ -0,0 +1,128 @@
+---
+title: V2.1
+weight: 55
+---
+
+# Version 2.1 release notes
+
+The Tempo team is pleased to announce the release of Tempo 2.1.
+
+With this release, Tempo enhances TraceQL so you can select traces based on kind, basic arithmetic functions, average/count, and numbers. In addition, Tempo 2.1 introduces vParquet2, a new experimental Parquet version designed to be more compatible with other Parquet implementations.
+
+>**Note**: For a complete list of changes and enhancements, refer to the [Tempo 2.1 changelog](https://github.com/grafana/tempo/releases).
+
+## Features and enhancements
+
+Some of the most important features and enhancements in Tempo 2.1 are highlighted below.
+
+### Enhancements to TraceQL
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. TraceQL makes it possible to find traces that otherwise would have been difficult or even impossible to identify with Tempo’s 1.x search.
+
+With this release, we’ve expanded the TraceQL language. In Tempo 2.0, TraceQL allowed users to select traces based upon resource attributes, time, duration, and basic aggregates. In Tempo 2.1, users can also select traces based on the following:
+
+* Enable querying by `kind`, an intrinsic attribute  (PR [2217](https://github.com/grafana/tempo/pull/2217))
```

Comment:
seeing a lot of double ` below. should they be single?

Reply:
Yes. Will fix. That was from the export app I used.